mean_shift_sklearn.py

```diff
@@ -44,21 +47,24 @@ def group_chrm(self, bed_file):
         """
         groups bed_file data(ndarray) w.r.t chromosome and strand for centroids
         >>> ms = Mean_Shift(3) # bandwidth:  3
-        >>> bd = [['chrX', '1', '+'], ['chrX', '2', '+'],
-        ... ['chrX', '3', '+'], ['chrX', '4', '+'],['chrX', '5', '+'],
-        ... ['chrX', '9', '-'], ['chrX', '10', '-'], ['chrX', '11', '-'],
-        ... ['chrX', '12', '-'], ['chrX', '13', '-'], ['chrY', '20', '+'],
-        ... ['chrY', '21', '+'], ['chrY', '22', '+'], ['chrY', '23', '+'],
-        ... ['chrY', '24', '+']]
+        >>> bd = [['chrX', '1', '+', '1'], ['chrX', '2', '+', '1'],
+        ... ['chrX', '3', '+', '1'], ['chrX', '4', '+', '1'],
+        ... ['chrX', '5', '+', '1'], ['chrX', '9', '-', '9'],
+        ... ['chrX', '10', '-', '9'], ['chrX', '11', '-', '9'],
+        ... ['chrX', '12', '-', '9'], ['chrX', '13', '-', '9'],
+        ... ['chrY', '20', '+', '20'], ['chrY', '21', '+', '20'],
+        ... ['chrY', '22', '+', '20'], ['chrY', '23', '+', '20'],
+        ... ['chrY', '24', '+', '20']]
         >>> sorted(ms.group_chrm(bd).items())
-        [(('chrX', '+'), ['1', '2', '3', '4', '5']),\
- (('chrX', '-'), ['9', '10', '11', '12', '13']),\
- (('chrY', '+'), ['20', '21', '22', '23', '24'])]
+        [(('chrX', '+', '1'), ['1', '2', '3', '4', '5']),\
+ (('chrX', '-', '9'), ['9', '10', '11', '12', '13']),\
+ (('chrY', '+', '20'), ['20', '21', '22', '23', '24'])]
         """
+
         chrm_details = dict()
 
         for b in bed_file:
-            chrm_name_strand = (b[0], b[2])
+            chrm_name_strand = (b[0], b[2], b[3])
```

Comment:
also please annotate what is in the selected columns

Reply:
chrm_name_strand is the columns? please name these according to what they are used for, e.g. key? and mention its content via comment